### PR TITLE
[WIP] ShowOnYes tests

### PR DESCRIPTION
**Pull request by dylanesque**
Initial test and setup for ShowOnYes component

### CODE REVIEW

File: src/components/AboutContent.js [hunk 1]
```diff
@@ -10,8 +10,7 @@ const AboutContent = function ({ snippets }) {
     <div>
 
 
-      <Header
-        className="ac-header"
+      <Header className="ac-header"
```
Comment (knod) [inline src/components/AboutContent.js]: I think it'd be best to avoid unrelated changes.

Comment (knod) [reply src/components/AboutContent.js]: If you do want to make some other changes, though, we can discuss them later.

**Comment by dylanesque** [comment]
corrupted repo
